galaxy/values-cvmfs.yaml

```diff
@@ -171,31 +171,28 @@ configs:
             </destination>
         </destinations>
     </job_conf>
-  galaxy.yml: |
+  galaxy.yml:
     uwsgi:
       virtualenv: /galaxy/server/.venv
       processes: 1
       http: 0.0.0.0:8080
-      static-map: /static/style=/galaxy/server/static/style/blue
-      static-map: /static=/galaxy/server/static
-      static-map: /favicon.ico=/galaxy/server/static/favicon.ico
       pythonpath: /galaxy/server/lib
       thunder-lock: true
       manage-script-name: true
-      mount: {{.Values.ingress.path}}=galaxy.webapps.galaxy.buildapp:uwsgi_app()
       buffer-size: 16384
```

Comment:
@almahmoud The code snippet in the configmap for converting to yaml `{{- $original := ($entry | toYaml) -}}` trips up on this line. Therefore, I made this a commandline argument to uwsgi too, but seems wrong.

Reply:
Fixed and made some more changes in: https://github.com/galaxyproject/galaxy-helm/pull/55/commits/dd3bd3495e9b6b26abac6a2a291180e3e4288e4e